react-app/package.json

```diff
@@ -4,15 +4,15 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.6.5",
-    "@cosmjs/amino": "0.28.4",
-    "@cosmjs/crypto": "0.28.4",
-    "@cosmjs/encoding": "0.28.4",
-    "@cosmjs/math": "0.28.4",
-    "@cosmjs/proto-signing": "0.28.4",
-    "@cosmjs/stargate": "0.28.4",
-    "@cosmjs/stream": "0.28.4",
-    "@cosmjs/tendermint-rpc": "0.28.4",
-    "@cosmjs/utils": "0.28.4",
+    "@cosmjs/amino": "^0.28.1",
+    "@cosmjs/crypto": "^0.28.1",
+    "@cosmjs/encoding": "^0.28.1",
+    "@cosmjs/math": "^0.28.1",
+    "@cosmjs/proto-signing": "^0.28.1",
+    "@cosmjs/stargate": "^0.28.1",
+    "@cosmjs/stream": "^0.28.1",
+    "@cosmjs/tendermint-rpc": "^0.28.1",
+    "@cosmjs/utils": "^0.28.1",
```

Comment:
oh sorry that's not what I meant, we specify `^0.28.4` but since `0.28.4` is part of `^0.28.1` so we don't need resolutions for dependencies with `^0.28.1` to use `0.28.4`

Reply:
oh I get now, I've updated